nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.js

```diff
@@ -32,15 +34,17 @@ import { Router } from '@angular/router';
  * @param nfRegistryApi     The api service.
  * @param changeDetectorRef     The change detector ref.
  * @param router                The angular router module.
+ * @param matDialog             The angular material dialog service
  * @constructor
  */
-function NfRegistry(http, nfStorage, nfRegistryService, nfRegistryApi, changeDetectorRef, router) {
+function NfRegistry(http, nfStorage, nfRegistryService, nfRegistryApi, changeDetectorRef, router, matDialog) {
     this.http = http;
     this.nfStorage = nfStorage;
     this.nfRegistryService = nfRegistryService;
     this.nfRegistryApi = nfRegistryApi;
     this.cd = changeDetectorRef;
     this.router = router;
+    this.matDialog = matDialog;
```

Comment:
OK to remove the service and its references and imports if it will not be used in `showAboutDialog` (see my comments about this method).